css/css-device-adapt/META.yml

```diff
@@ -0,0 +1,2 @@
+suggested_reviewers:
+  - ChumpChief
```

Comment:
I've poked @ChumpChief but got no reaction. Luke, let's just put ourselves here. If implementer interest for this spec materializes (it was implemented in Presto, FWIW) we can hand it off.